editor/Makefile

```diff
@@ -90,4 +106,6 @@ test : all
 
 clean :
 	rm -rf gen
+	rm -rf ketcher/node_modules
+	rm -rf ketcher/dist
```

Comment:
If ketcher/dist was not created by make, then I would not delete it with "make clean".